deebot_client/hardware/deebot/2pv572.py

```diff
@@ -86,13 +82,6 @@
         ),
         network=CapabilityEvent(NetworkInfoEvent, []),
         play_sound=CapabilityExecute(PlaySound),
-        settings=CapabilitySettings(
```

Comment:
I think the settings key is required:

```
  File "client.py/deebot_client/hardware/deebot/2pv572.py", line 55, in <module>
    Capabilities(
    ~~~~~~~~~~~~^
        availability=CapabilityEvent(AvailabilityEvent, []),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<33 lines>...
        ),
        ^^
    ),
    ^
TypeError: Capabilities.__init__() missing 1 required keyword-only argument: 'settings'
```

Reply:
@flubshi That's correct but we do not support any of those settings yet. In particular the volume seems mandatory.
I'm going to push a "fake volume" command tomorrow so that both HA and the library are happy :)

Reply:
The XML robot Deebot 900 has also no volume configuration. Maybe we should make it optional in the library instead of a fake module.